assets/js/lib/test-utils/factories.js

```diff
@@ -87,3 +87,9 @@ export const hostnameFactory = Factory.define(() => ({
   id: faker.datatype.uuid(),
   hostname: faker.hacker.noun(),
 }));
+
+export const aboutPageFactory = Factory.define(() => ({
```

Comment:
I would name this `aboutPage`, simply `aboutFactory` as it returns the about data, even though it is only used in the about page by now hehe

Reply:
Yeah thanks!
Naming is hard haha